pkg/pf/tfbridge/provider_delete.go

```diff
@@ -52,13 +53,17 @@ func (p *provider) DeleteWithContext(
 		return resource.StatusOK, err
 	}
 
+	plannedState, err := tfprotov6.NewDynamicValue(tfType, tftypes.NewValue(tfType, nil))
+	contract.AssertNoErrorf(err, "nil is always a valid value to marshal to a dynamic state")
```

Comment:
Did you mean dynamic value?

Reply:
No, I meant the `nil` in `tftypes.NewValue(tfType, nil)`.

Reply:
The error says "dynamic state" not "dynamic value" - what is dynamic state? @iwahbe

Reply:
Implicitly, a dynamic value that is treated as state, in this case planned state.